runtime/doc/builtin.txt

```diff
@@ -6957,10 +6957,8 @@ sha256({string})                                                      *sha256()*
 shellescape({string} [, {special}])                              *shellescape()*
 		Escape {string} for use as a shell command argument.
 
-		On Windows when 'shellslash' is not set, encloses {string} in
-		double-quotes and doubles all double-quotes within {string}.
-		Otherwise encloses {string} in single-quotes and replaces all
-		"'" with "'\''".
+		On Windows, encloses {string} in single-quotes and replaces
+		all "'" with "'\''".
```

Comment:
This is not correct. If the shell is cmd.exe, double quotes should be used (and "doubles all double-quotes"). shellescape() inherently depends on 'shell'.